torch/_refs/nn/functional/__init__.py

```diff
@@ -59,6 +61,58 @@
 
 Tensor = torch.Tensor
 
+
+# TODO: should we allow the user to set a different dtype for the mask generation?
+def _dropout_helper(
```

Comment:
Add a comment (or docstring) describing what this function does

Reply:
Sure. Added. Thanks!